### PR TITLE
update change log for 0.6.1

### PR DESCRIPTION
**Pull request by cshanahan1**

### CODE REVIEW

**Review by kmacdonald-stsci — APPROVED** [review]

File: CHANGES.rst [hunk 1]
```diff
@@ -1,4 +1,7 @@
-0.6.1 (unreleasd)
+0.6.2 (unreleased)
+================
```
Comment (hbushouse) [inline CHANGES.rst:2]: parser might barf because the line of "=" characters is not as long as the text above it.

**Review by mcara — APPROVED** [review]
LGTM

**Review by mcara — CHANGES_REQUESTED** [review]
@hbushouse is right. Make sure length of `====` is the same as the text above. See https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#sections:

> Section headers are created by underlining (and optionally overlining) the section title with a punctuation character, **at least as long as the text**

**Review by ddavis-stsci — APPROVED** [review]
Even I feel like I can approve this one.

**Review by mcara — APPROVED** [review]

**Review by hbushouse — APPROVED** [review]

**Comment by mcara** [comment]
I think @nden is on the fence

**Pull request merged**